Cargo.toml

```diff
@@ -50,3 +50,9 @@ git = "https://github.com/launchbadge/sqlx/"
 branch = "master"
 default-features = false
 features = ["runtime-actix", "postgres", "chrono", "offline"]
+
```

Comment:
Formatting;  This should probably be a single newline, and there should be a newline at the end of the file for consistency.